### PR TITLE
Add simpleGemmAlphaVecExt

### PR DESCRIPTION
**Pull request by KKyang**

### CODE REVIEW

File: clients/samples/gemm_alphavec/sample_hipblaslt_gemm_alphavec_ext.cpp [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    if(heuristicResult.empty())
+    {
+        std::cerr << "No valid solution found!" << std::endl;
+        return;
+    }
+
+    // In this sample, the workspace is already allocated with max_workspace_size
+    // If not, calculate the needed workspace_size and allocate d_workspace here
+    // uint64_t workspace_size = 0;
+    // for(int i = 0; i < returnedAlgoCount; i++)
```
Comment (jichangjichang) [inline clients/samples/gemm_alphavec/sample_hipblaslt_gemm_alphavec_ext.cpp:139]: add check if the d_worksapce size is biger enough for returned algo

File: clients/samples/gemm_alphavec/sample_hipblaslt_gemm_alphavec_ext.cpp [hunk 2]
```diff
@@ -0,0 +1,147 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (C) 2022-2023 Advanced Micro Devices, Inc.
```
Comment (jichangjichang) [inline clients/samples/gemm_alphavec/sample_hipblaslt_gemm_alphavec_ext.cpp:5]: for new one should be 2023

Comment (KKyang) [reply clients/samples/gemm_alphavec/sample_hipblaslt_gemm_alphavec_ext.cpp]: Fixed

File: clients/samples/gemm_alphavec/sample_hipblaslt_gemm_alphavec_ext.cpp (hunk 1 above)
Comment (KKyang) [reply clients/samples/gemm_alphavec/sample_hipblaslt_gemm_alphavec_ext.cpp:139]: Fixed

**Review by jichangjichang — APPROVED** [review]
LGTM

**Pull request merged**